dbms/src/Storages/DeltaMerge/File/DMFileReader.h

```diff
@@ -153,23 +131,18 @@ class DMFileReader
     const bool enable_del_clean_read;
     const bool is_fast_scan;
 
-    const UInt64 max_read_version;
+    const bool enable_column_cache;
```

Comment:
This is a very important point. I'd like to  examine layout other stackable classes in Storages module like `Region` to reduce padding between fields. https://github.com/pingcap/tiflash/issues/8758